Source/CoverageConfiguration.pas

```diff
@@ -405,21 +416,8 @@ function TCoverageConfiguration.UseTestExePathAsWorkingDir: Boolean;
 function TCoverageConfiguration.IsPathInExclusionList(const APath: TFileName): Boolean;
 var
   Mask: string;
-  IsIncluded: boolean;
 begin
   Result := False;

```

Comment:
You have removed functionality and (probably) broke existing working. Please change in such a way that it keeps backwards compatible.

Reply:
Fixed in last commit